libcontainer/configs/mount_linux.go

```diff
@@ -15,6 +15,10 @@ type Mount struct {
 	// Mount flags.
 	Flags int `json:"flags"`
 
+	// Mount flags that were explicitly cleared in the configuration (meaning
+	// the user explicitly requested that these flags *not* be set).
```

Comment:
Is that proper english? Not a native speaker, so I might e wrong, but I think this should be correct (feel free to ignore if I'm wrong :)):


```suggestion
	// the user explicitly requested that these flags are *not* set).
```

Reply:
"not be set" is proper English here. The difference is that "request these flags not be set" means to request the flags to not be set. "Request the flags are not set" sounds a little weird to me because "are not set" is not a request, it's a state.